DSI/globus_gridftp_server_iRODS.cpp

```diff
@@ -2004,6 +2009,8 @@ globus_l_gfs_iRODS_recv(
     globus_gfs_transfer_info_t *        transfer_info,
     void *                              user_arg)
 {
+    namespace fs = irods::experimental::filesystem;
```

Comment:
Is this needed? It doesn't appear to be used anywhere.

Reply:
No, not anymore.  I will remove it.